docs/app/templates/docs/advanced-markers.hbs

```diff
@@ -0,0 +1,42 @@
+<div class="row">
+  <div class="col-lg-7 order-2 order-lg-1">
+    <section>
+      <h5 id="markers">Creating advanced markers</h5>
+
+      <p>You can create an advanced marker with the yielded <var>advancedMarker</var> component. Once again, you can use either <var>lat</var> and <var>lng</var>, or <var>position</var>, to set the position.</p>
```

Comment:
It would be great to mention here that these are now preferred over the old marker. And the old marker docs should mention the deprecation and link to this page.